tutorials/R/download-ais-data/download-NEON-AIS-data.Rmd

```diff
@@ -0,0 +1,307 @@
+---
+syncID: 
+title: "Download and work with NEON Aquatic Instrument Data "
+description: Tutorial for downloading NEON AIS data using the neonUtilities package and then exploring and understanding the downloaded data including seperating out into collection locations using the HOR variable. 
+dateCreated: '2020-05-19'
+authors: Bobby Hensley, Megan Jones
+contributors: 
+estimatedTime: 
+packagesLibraries: neonUtilities, ggplot2
+topics: data-management, rep-sci
+languageTool: R, API
+code1: R/download-explore/download-NEON-AIS-data.R
+tutorialSeries:
+urlTitle: download-neon-ais-data
+---
+
+This tutorial covers downloading NEON aquatic instrument system (AIS) data, using the 
+neonUtilities R package, as well as basic instruction in beginning to 
+explore and work with the downloaded data, including guidance in 
+navigating data documentation and understanding how to seperate data using the 
+horizontal (HOR) variable. 
+
+To set up our mini analyis in which we learn to access and work with the NEON AIS 
+data, our goal is to create a seperate R object for data from the various sensor
+locations and to create a plot of a variable of interest to compare the data from 
+different sensors. 
+
+<div id="ds-objectives" markdown="1">
+
+## Objectives
+
+After completing this activity, you will be able to:
+
+* Download NEON AIS data using the neonUtilities package.
+* Understand downloaded data sets and load them into R for analyses.
+* Seperate data collected at different sensor locations using the HOR variable.
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
```

Comment:
Update from "most current version of R" to R, version >3.4

Reply:
Ha! You'd think I could remember to do this on my own tutorial.